apps/sms/services/js/message_manager.js

```diff
@@ -41,28 +53,64 @@ var MessageManager = {
     );
   },
 
+  /**
+   * Event occurs when an SMS or MMS enters the send flow.
+   * @param {MozSmsEvent|MozMmsEvent} e Instance of
+   * {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}
+   */
   onMessageSending: function mm_onMessageSending(e) {
     Threads.registerMessage(e.message);
 
     this.emit('message-sending', { message: e.message });
   },
 
+  /**
+   * Specifies an event listener to receive failed events. These events occur
+   * when an SMS or MMS has failed.
+   * @param {MozSmsEvent|MozMmsEvent} e Instance of
+   * {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}
+   */
   onMessageFailed: function mm_onMessageFailed(e) {
     this.emit('message-failed-to-send', { message: e.message });
   },
 
+  /**
+   * Specifies an event listener to receive deliverysuccess events. These events
+   * occur when an SMS or MMS is delivered.
+   * @param {MozSmsEvent|MozMmsEvent} e Instance of
+   * {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}
+   */
   onDeliverySuccess: function mm_onDeliverySuccess(e) {
     this.emit('message-delivered', { message: e.message });
   },
 
+  /**
+   * Specifies an event listener to receive received events. These events occur
+   * when an SMS or MMS is received.
+   * @param {MozSmsEvent|MozMmsEvent} e Instance of
+   * {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}
+   */
   onReadSuccess: function mm_onReadSuccess(e) {
     this.emit('message-read', { message: e.message });
   },
 
+  /**
+   * Specifies an event listener to receive sent events. These events occur when
+   * an SMS or MMS is sent.
+   * @param {MozSmsEvent|MozMmsEvent} e Instance of
+   * {@link http://mdn.io/MozSmsEvent} or {@link http://mdn.io/MozMmsEvent}
+   */
   onMessageSent: function mm_onMessageSent(e) {
     this.emit('message-sent', { message: e.message });
   },
 
+  /**
+   * Specifies an event listener to receive received events. These events occur
+   * when an SMS or MMS is received.Event handler executed when new message is
```

Comment:
__nit:__ looks like last sentence is redundant. But honestly I like last description style more :) - ```Event handler that's executed when new SMS or MMS is received``` - short and clear - I'd use such style for all handler descriptions, but won't block on it - up to you.